frontend/src/features/admin-form/create/end-page/EndPageDrawer.tsx

```diff
@@ -210,6 +225,10 @@ export const EndPageDrawer = (): JSX.Element | null => {
     ),
   )
 
+  const isPaymentEnabled =
+    form?.responseMode === FormResponseMode.Encrypt &&
+    form?.payments_field.enabled
```

Comment:
```suggestion
    form?.payments_channel.channel !== PaymentChannel.Unconnected
```
Will need to update this (along with import PaymentChannel) once the payment enablement PR is merged in